src/aws.ts

```diff
@@ -440,7 +440,7 @@ export namespace $AWS {
       operationName: Op,
       integration: Omit<
         Integration<F, `$AWS.DynamoDB.${Op}`>,
-        "kind" | "native"
+        "kind" | "native" | "__functionBrand"
```

Comment:
DO we need a type alias for this behavior? Why do these fields need to be omitted?